code/SQLite3SchemaManager.php

```diff
@@ -210,7 +210,7 @@ public function checkAndRepairTable($tableName = null)
     {
         $ok = true;
 
-        if (!SapphireTest::using_temp_db() && !self::$checked_and_repaired) {
+        if (!SapphireTest::getTempDB()->isUsed() && !self::$checked_and_repaired) {
```

Comment:
We need to remove SapphireTest references from non-test code. This entire method will fail if phpunit isn't installed, which is often the case in production.

Reply:
You can just use `TempDatabase::singleton()->isUsed()` if necessary here.

Reply:
Thanks. Does #35 make this PR unnecessary?

Reply:
Yes I think so. :)